frontend/src/app/shared/osm/osm-address/osm-address.component.ts

```diff
@@ -38,18 +38,22 @@ export class OsmAddressComponent implements OnInit, ControlValueAccessor, Valida
     { value: 'Österreich', viewValue: 'common.austria' }
   ];
 
-
   addressForm = this.fb.group({
     id: [''],
     name: [''],
     description: [''],
-    country: [this.countries[0].viewValue, Validators.required],
-    street: ['', Validators.pattern('^[a-zA-ZäüöÄÜÖß.-]*')],
-    streetNumber: ['', Validators.pattern('^[a-zA-Z0-9äüöÄÜÖß.-]*')],
+    country: [this.countries[0].value, Validators.required],
+    street: ['', Validators.pattern('[a-zA-Z äüöÄÜÖß.-]{1,}')],
+    streetNumber: ['', Validators.pattern('[a-zA-Z0-9äüöÄÜÖß.-]{1,}')],
     zipcode: ['', Validators.pattern('^[0-9]{4,5}$')],
     city: ['', Validators.pattern('^[a-zA-ZäüöÄÜÖß.-]*')],
   });
 
+  isValid():boolean {
+    return this.addressForm.get('country').value.length > 0 && this.addressForm.get('street').value.length > 0
+      && this.addressForm.get('streetNumber').value.length > 0 && this.addressForm.get('zipcode').value.length > 0
+      && this.addressForm.get('city').value.length > 0 && !this.addressForm.get('street').value.startsWith(' ');
+  }
```

Comment:
Auch die Validierungen gehören alle in die FormGroup, damit Pflichtfelder für den User sichtbar sind.
Aktuell sieht es so aus, als ob nur Vorname und Nachname Pflichtfelder sind:

![image](https://user-images.githubusercontent.com/60063612/103654299-fc699080-4f65-11eb-8329-d08dfa417b11.png)

Für die Straße (startsWith(' ')) kannst du einen eigenen Validator schreiben, z.B. so:

```
private noWhitespaceValidator(control: FormControl) {
		const beginsWithWhitespace = (control.value || '').startsWith(' ');
		return beginsWithWhitespace ? { 'whitespace': true } : null;
}
```

Reply:
vielen dank!!